jerry-core/vm/vm.c

```diff
@@ -1216,50 +1338,22 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           JERRY_ASSERT (*byte_code_p >= CBC_CALL && *byte_code_p <= CBC_CALL2_PROP_BLOCK);
           continue;
         }
-        case VM_OC_CALL_N:
-        case VM_OC_CALL_PROP_N:
-        {
-          right_value = (unsigned int) ((opcode - CBC_CALL0) / 6);
-          /* FALLTHRU */
-        }
         case VM_OC_CALL:
-        case VM_OC_CALL_PROP:
         {
-          ecma_value_t this_value = ecma_make_simple_value (ECMA_SIMPLE_VALUE_UNDEFINED);
+          JERRY_ASSERT (free_flags == 0);
 
-          stack_top_p -= right_value;
-
-          if (VM_OC_GROUP_GET_INDEX (opcode_data) >= VM_OC_CALL_PROP_N)
+          if (!frame_ctx_p->call_operation)
```

Comment:
This is a bit hard to read. Is this condition the same as `frame_ctx_p->call_operation == VM_NO_EXEC_OP`?